server/python/src/routers/movies.py

```diff
@@ -1,5 +1,5 @@
 from fastapi import APIRouter, Query, Path, Body
-from src.database.mongo_client import db, get_collection
+from src.database.mongo_client import db, get_collection, voyage_ai_available
 from src.models.models import VectorSearchResult, CreateMovieRequest, Movie, MovieFilter, SuccessResponse, UpdateMovieRequest, SearchMoviesResponse, BatchUpdateRequest, BatchDeleteRequest
```

Comment:
Looks like these are unused imports. Is this functionality we haven't implemented yet, or functionality that we're not handling here in the router?

```suggestion
from src.models.models import VectorSearchResult, CreateMovieRequest, Movie, SuccessResponse, UpdateMovieRequest, SearchMoviesResponse
```